src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -351,6 +353,7 @@ HRESULT STDMETHODCALLTYPE CorProfiler::ModuleLoadFinished(ModuleID module_id,
       "Microsoft.Extensions.Options"_W,
       "Microsoft.Extensions.ObjectPool"_W,
       "System.Configuration"_W,
+      "System.Web"_W,
```

Comment:
While you're in here, should we add the extremely common `System.Xml` to this list?

Reply:
Addressed in 60ad318fb3fbe2d48b3477163178326edb021cc3